### DISCUSSION TITLE
can not read my custom rules.

### DISCUSSION THREAD

**Discussion by hirasaki1985** [eslint/eslint#15416] [Q&A] [answered]
I'm creating custom plugin with eslint.
I finished to develop this, and tested it.

then, I installed my custom plugin in the another repository with npm command.
and add rules to .eslint.js.

This another repository can find my custom plugin name 'coding-rules'.
but can not find custom rules name 'no-use-specific-imports'.

repository is here.
https://github.com/hirasaki1985/eslint-plugin-coding-rules

I can't solve it. please help me.

**Comment by aladdin-add** [comment, marked as answer]
the ruleId should be `coding-rules/no-use-specific-imports`.

**Comment by hirasaki1985** [comment]
@aladdin-add 
thank you for reply. I'll try it.

**Reply by hirasaki1985** [reply]
I was able to solve it. and released my custom eslint plugin module.
thank you.
